application/deed/deed_service.py

```diff
@@ -0,0 +1,98 @@
+import logging
```

Comment:
I agree with the refactor, ironically this application is starting to look more and more like ALPHA as we develop it. Suggestion, should we just call it service as its in the deed folder?